Getting-Started/Data/Members/index.md

```diff
@@ -1,10 +1,10 @@
 # Members
-*Members are used for registering and authenticating external users of an Umbraco installation (ie. forum members, intranet users and so forth). Unlike with Document Types and Media Types everything is done in the Members section both defining and creating, and editing members. This guide will explain how to define and create members in the backoffice. If you want to work with members using the service APIs, links can be found at the end of document*
+*Members are used for registering and authenticating external users of an Umbraco installation (ie. forum members, intranet users and so forth). Unlike with Document Types and Media Types everything is done in the Members section both defining and creating, and editing members. This guide will explain how to define and create members in the back office. If you want to work with members using the service APIs, links can be found at the end of the document*
```

Comment:
'backoffice' is the official word for it in Umbraco terminology, so it should be in one word 😉

Reply:
Ah, thank you for the heads up!
I have corrected this is my master - is this sufficient for the PR to be merged?
Thanks again 🙂